raft/server.cc

```diff
@@ -645,7 +654,7 @@ future<entry_id> server_impl::add_entry_on_leader(command cmd, seastar::abort_so
         try {
             memory_permit = co_await _fsm->wait_for_memory_permit(as, log::memory_usage_of(cmd, _config.max_command_size));
         } catch (semaphore_aborted&) {
-            throw request_aborted();
+            throw request_aborted(format("Semaphore aborted while adding entry on leader in term: {}, on server: {}", t, id()));
```

Comment:
Here you could be more specific, that we were waiting for available memory before adding entry.

Reply:
Also you used `_id` instead of `id()` in other places. Pick one and stick to it for consistency